scripts/shopify-tag.css

```diff
@@ -74,12 +74,12 @@ p {
     top: 0;
     left: 0;
     opacity: 50%;
-    z-index: 100;
+    z-index: 2000;
     backdrop-filter: brightness(0.5);
 }
 
-#hc-store-information {
```

Comment:
what was max height doing before and why is it ok to remove it now?

Reply:
We are using it to have a fixed height of store-info div, previously it does not have any effect as we have declared `hc-store-information` as a class but using it as id, so the max-height was not even applying, so removed it and used class selector.

Reply:
got it